src/components/ReportStatus.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect } from "react";
+import { Formik, Form } from "formik";
+import { Link } from 'react-router-dom';
+import { GetResponseErrors } from "../utilities/CitysourcedResponseHelpers";
+import { GetReportComments } from '../services/authService';
+import FormContainer from './FormContainer';
+import Note from './Note';
+import Moment from 'react-moment';
+import SeButton from "./SeButton";
+import { Go, Routes } from "../Routing";
+
+
+const ReportStatus = (props, routeProps) => {
+	const { trackingNumber } = props.values;
+	const response = props.history.location.state;
+	const [comments, setComment] = useState([]);
+	const [commentLength, setcommentLength] = useState([]);
+
+	let alertMessage = '';
+	let errorStatusCode ='';
+	let reportId, reportDateCreated, reportDateUpdated, reportRequestType, address, status, isOpen;
+
+	useEffect(() => {
+		setcommentLength(3);
```

Comment:
this line can be moved to the declartion.

```js
const [commentLength, setcommentLength] = useState(3);
```

this also makes it clear that this is a number, and not an array.

Reply:
changed to number declartion along with the new variable to set the name of the "link" ("Show more...")